spec/features/traject/ead2_indexing_spec.rb

```diff
@@ -75,6 +81,11 @@
           expect(result['components'].first[field]).to include 'Stanford University Libraries. Special Collections and University Archives'
         end
       end
+      it 'geogname' do
+        %w[geogname_sim geogname_ssm].each do |field|
+          expect(result['components'].first[field]).to be_nil
```

Comment:
This fixture has a `geogname` at the collection level but not at the component level.